app/models/user.rb

```diff
@@ -68,8 +68,8 @@ def can?( capability_name, category_name = :general )
     cc = CapabilityCategory.find_by( name: category_name.to_s )
     return true if capabilities.exists? name: capability_name.to_s, category: cc
 
-    Rails.logger.debug  "Capability check failed: '#{username}' " \
-                        "cannot '#{capability_name}' '#{category_name}'"
+    Rails.logger.debug  'Capability check failed: ' \
```

Comment:
All models (and mailers and controllers) have a `#logger` method that returns `Rails.logger`. The `Rails.` can be dropped from line 71. https://guides.rubyonrails.org/debugging_rails_applications.html#sending-messages